### ISSUE TITLE
When using SoftCrudRepository can't use relations in findById

### ISSUE THREAD

**Issue by SandroMoretti** [sourcefuse/loopback4-soft-delete#25]
When using SoftCrudRepository can't use relations in findById.
The find(filter, filter) works fine. But findById(id, filter) don't work.

**To Reproduce**

1. Create an model with BelongsTo or any relation
2. Extends the repository with SoftCrudRepository
3. In the controller use find({include: {relation: ['RELATION_NAME']}} -> this will work
4. In the controller use findById(id, {include: {relation: ['RELATION_NAME']}} -> this will not work

**Comment by SandroMoretti** [comment]
I have tested and this happen due the follow lines of file: ```soft-crud.repository.base.js```


![image](https://user-images.githubusercontent.com/6597833/114224944-1f7e5480-9948-11eb-8a61-0bdc36822450.png)


To prevent undefined/null, the plugin test if is an valid filter, if dont filter = {} (an new filter)
But in next line the plugin just ignore the validation in last line and create an new filter. This replace the old filter that contains the include.

I changed to:

![image](https://user-images.githubusercontent.com/6597833/114225160-5fddd280-9948-11eb-9184-2c49b23d4d6b.png)



Then works fine.
I'll try to do an pull request, but I'm newbie with TS and NPM, even that I can modify the .js file I don't know if I'll get do the trick with TS  in the right place.

**Comment by SandroMoretti** [comment]
I checked the ts code and its everything fine.

```
![image](https://user-images.githubusercontent.com/6597833/114227215-104cd600-994b-11eb-8232-9df22dd1ba35.png)
```

This is the branch master part of code. I made a copy of this file and test with then separeted of plugin, and evertying works.

My plugin is in the same version of master branch:

![image](https://user-images.githubusercontent.com/6597833/114227321-3a9e9380-994b-11eb-942c-06f984930c97.png)



I don't know why my .js file its with that bug.
Someone can explain?

I'll use a copy of soft-crud.repository.base.ts while this.
